treesapp_assign_and_update.Rmd

```diff
@@ -106,36 +150,40 @@ treesapp update \
   --delete \
   --cluster \
   --trim_align \
+  --min_taxonomic_rank o \
   -n 4 \
-  -m prot \
-  --output XmoA_FunGene_update/ \
+  --output XmoA_UniProt_update/ \
   --skip_assign \
-  --treesapp_output p_amoA_FunGene9.5_isolates_assign/ \
-  --refpkg_path XmoA_seed/final_outputs/XmoA_build.pkl
+  --treesapp_output UniProt_XmoA_assign/ \
+  --refpkg_path XmoA_seed/final_outputs/XmoA_build.pkl \
+  --seqs2lineage UniProt_XmoA.tsv
```

Comment:
could be just me, but I had to filter out rows with `SCGC` first.
ex. `Archaea	Crenarchaeota	crenarchaeote SCGC AAA288-N08`

Reply:
Hmm, could you share your `UniProt_XmoA.tsv` file with me? My guess is our searches in UniProt were slightly different and I didn't have this archaeum in my dataset. Maybe it would be wise to include the shell command you needed to run in a footnote.